misc/test/examples_test.go

```diff
@@ -455,133 +455,148 @@ func TestExamples(t *testing.T) {
 
 	longTests := []integration.ProgramTestOptions{
-		//base.With(integration.ProgramTestOptions{
-		//	Dir: path.Join(cwd, "..", "..", "azure-ts-aks-helm"),
-		//	Config: map[string]string{
-		//		"azure:environment": azureEnviron,
-		//		"password":          "testTEST1234+_^$",
-		//		"sshPublicKey":      "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDeREOgHTUgPT00PTr7iQF9JwZQ4QF1VeaLk2nHKRvWYOCiky6hDtzhmLM0k0Ib9Y7cwFbhObR+8yZpCgfSX3Hc3w2I1n6lXFpMfzr+wdbpx97N4fc1EHGUr9qT3UM1COqN6e/BEosQcMVaXSCpjqL1jeNaRDAnAS2Y3q1MFeXAvj9rwq8EHTqqAc1hW9Lq4SjSiA98STil5dGw6DWRhNtf6zs4UBy8UipKsmuXtclR0gKnoEP83ahMJOpCIjuknPZhb+HsiNjFWf+Os9U6kaS5vGrbXC8nggrVE57ow88pLCBL+3mBk1vBg6bJuLBCp2WTqRzDMhSDQ3AcWqkucGqf dremy@remthinkpad",
-		//	},
-		//	ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
-		//		assertHTTPResult(t, stack.Outputs["serviceIP"], func(body string) bool {
-		//			return assert.Contains(t, body, "It works!")
-		//		})
-		//	},
-		//}),
-		//// TODO: This test fails due to a bug in the Terraform Azure provider in which the
-		//// service principal is not available when attempting to create the K8s cluster.
-		//// See the azure-ts-aks-multicluster readme for more detail and
-		//// https://github.com/terraform-providers/terraform-provider-azurerm/issues/1635.
-		//// base.With(integration.ProgramTestOptions{
-		////	Dir: path.Join(cwd, "..", "..", "azure-ts-aks-multicluster"),
-		////	Config: map[string]string{
-		////		"azure:environment": azureEnviron,
-		////		"password":          "testTEST1234+_^$",
-		////		"sshPublicKey":      "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDeREOgHTUgPT00PTr7iQF9JwZQ4QF1VeaLk2nHKRvWYOCiky6hDtzhmLM0k0Ib9Y7cwFbhObR+8yZpCgfSX3Hc3w2I1n6lXFpMfzr+wdbpx97N4fc1EHGUr9qT3UM1COqN6e/BEosQcMVaXSCpjqL1jeNaRDAnAS2Y3q1MFeXAvj9rwq8EHTqqAc1hW9Lq4SjSiA98STil5dGw6DWRhNtf6zs4UBy8UipKsmuXtclR0gKnoEP83ahMJOpCIjuknPZhb+HsiNjFWf+Os9U6kaS5vGrbXC8nggrVE57ow88pLCBL+3mBk1vBg6bJuLBCp2WTqRzDMhSDQ3AcWqkucGqf dremy@remthinkpad",
-		////	},
-		//// }),
+		base.With(integration.ProgramTestOptions{
+			Dir: path.Join(cwd, "..", "..", "azure-ts-aks-helm"),
+			Config: map[string]string{
+				"azure:environment": azureEnviron,
+				"password":          "testTEST1234+_^$",
+				"sshPublicKey":      "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDeREOgHTUgPT00PTr7iQF9JwZQ4QF1VeaLk2nHKRvWYOCiky6hDtzhmLM0k0Ib9Y7cwFbhObR+8yZpCgfSX3Hc3w2I1n6lXFpMfzr+wdbpx97N4fc1EHGUr9qT3UM1COqN6e/BEosQcMVaXSCpjqL1jeNaRDAnAS2Y3q1MFeXAvj9rwq8EHTqqAc1hW9Lq4SjSiA98STil5dGw6DWRhNtf6zs4UBy8UipKsmuXtclR0gKnoEP83ahMJOpCIjuknPZhb+HsiNjFWf+Os9U6kaS5vGrbXC8nggrVE57ow88pLCBL+3mBk1vBg6bJuLBCp2WTqRzDMhSDQ3AcWqkucGqf dremy@remthinkpad",
+			},
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				assertHTTPResult(t, stack.Outputs["serviceIP"], func(body string) bool {
+					return assert.Contains(t, body, "It works!")
+				})
+			},
+		}),
+		// TODO: This test fails due to a bug in the Terraform Azure provider in which the
```

Comment:
Does this test need to be disabled?

Reply:
oh good question - these are the "long" tests and are not run anywhere yet - this was because I disabled it in the last PR by mistake :D

These tests timeout 100% of the time because of that Azure provider bug